src/app/layout.tsx

```diff
@@ -5,7 +5,8 @@ import "./globals.css";
 const raleway = Raleway({ subsets: ["latin"] });
 
 export const metadata = {
-	description: "Find your npm packages eligible for Tidelift funding. 💸",
+	description:
```

Comment:
I just want to note that the prettier check doesn't pass if I have the description property all on one line, as in:

````js
export const metadata = {
	description: "Check if your npm packages are eligible for Tidelift funding. 💸<",
	title: "Tidelift Me Up",
};
````

This is why the description property code is written/formatted this way.

Reply:
Yip yip, this is expected and 👍